From 05de6e1b52a809e34d9124dc2a094759613d2d4c Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Wed, 24 May 2017 17:05:52 +0800
Subject: [PATCH 1938/5242] MLK-15001-37 arm64: dtsi: fsl-imx8qxp: Add
 pd_mipi_dsi1 support

commit  37f0a79fe34e6c254f63056a105ce49c31c4a0e0 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch adds pd_mipi_dsi1 support.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |   27 ++++++++++++++++++++++++
 1 file changed, 27 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index dbdd411..6ffd73b 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -632,6 +632,33 @@
 			};
 		};
 
+		pd_mipi_dsi1: PD_MIPI_1_DSI {
+			compatible = "nxp,imx8-pd";
+			reg = <SC_R_MIPI_1>;
+			#power-domain-cells = <0>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+
+			pd_mipi_dsi_1_i2c0: PD_MIPI_1_DSI_I2C0 {
+				name = "mipi1_dsi_i2c0";
+				reg = <SC_R_MIPI_1_I2C_0>;
+				#power-domain-cells = <0>;
+				power-domains =<&pd_mipi_dsi1>;
+			};
+			pd_mipi_dsi_1_i2c1: PD_MIPI_1_DSI_I2C1 {
+				name = "mipi1_dsi_i2c1";
+				reg = <SC_R_MIPI_1_I2C_1>;
+				#power-domain-cells = <0>;
+				power-domains =<&pd_mipi_dsi1>;
+			};
+			pd_mipi_1_pwm0: PD_MIPI_1_DSI_PWM0 {
+				name = "mipi1_dsi_pwm0";
+				reg = <SC_R_MIPI_1_PWM_0>;
+				#power-domain-cells = <0>;
+				power-domains =<&pd_mipi_dsi1>;
+			};
+		};
+
 		pd_isi_ch0: PD_ISI_CH0 {
 			compatible = "nxp,imx8-pd";
 			reg = <SC_R_ISI_CH0>;
-- 
1.7.9.5

